a/airflow/airflow_2.0.1.sh

```diff
@@ -0,0 +1,123 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : Airflow
+# Version       : 2.0.1
+# Source repo   : https://github.com/apache/airflow
+# Tested on     : rhel_7.9
+# Script License: Apache License Version 2.0
+# Maintainer    : Swati Singhal <swati.singhal@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+CWD=`pwd`
+export PATH=${PATH}:$HOME/conda/bin
+export PATH=${PATH}:/root/conda/envs/airflow/bin
+export PYTHONPATH=${PYTHONPATH}:/root/conda/envs/airflow/lib/python3.8/site-packages
+export PYTHON_VERSION=3.8
+export AIRFLOW_HOME=~/airflow
+
+export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
+export CRYPTOGRAPHY_DONT_BUILD_RUST=1
+export PYTHON=/usr/local/bin/python3.8
+
+#Core test case essentials
+export AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__VISIBILITY_TIMEOUT=21600
+export AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS___TEST_ONLY_BOOL=true
+export AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS___TEST_ONLY_FLOAT=1.6
+export AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS___TEST_ONLY_STRING=this is a test
+
+dnf -y --disableplugin=subscription-manager install http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/Packages/centos-linux-repos-8-2.el8.noarch.rpm https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
Any specific reason using CentOS repos instead of RHEL repos?

Please check http://rhn.pbm.ihost.com/rhn/latest/ , if packages you need can be be found in any of the listed repos for RHEL.

Reply:
Unable to build using rhn repositories. was facing errors.